oneanddone/tasks/models.py

```diff
@@ -367,7 +374,7 @@ class Task(CachedModel, CreatedModifiedModel, CreatedByModel):
     instructions = models.TextField()
     is_draft = models.BooleanField(verbose_name='draft')
     is_invalid = models.BooleanField(verbose_name='invalid')
-    name = models.CharField(max_length=255, verbose_name='title', unique=True)
```

Comment:
I assume this was a mistake. We don't want to remove the `unique` argument from this `CharField`.